package.json

```diff
@@ -39,13 +39,15 @@
     "html-minifier": "~4.0.0",
     "klaw": "~4.0.1",
     "klaw-sync": "~6.0.0",
+    "link": "^1.4.0",
```

Comment:
```suggestion
    "link": "~1.4.0",
```

Reply:
What is this dependency anyway?

Reply:
I think he added it by accident. You should remove this.